src/patterns/OcMenu.vue

```diff
@@ -6,7 +6,7 @@
     <div uk-dropdown="mode: click, offset: 0">
       <!-- @slot The default slot for inner content  -->
       <slot name="default"> </slot>
-      <ul class="uk-nav uk-dropdown-nav uk-nav-default">
+      <ul class="uk-list uk-margin-remove-bottom">
```

Comment:
The design of the component is "too narrow" in the sense of flexibility. Better to create a separate `oc-nav` component and so it can be mixed in in the slot where needed.

Reply:
No ul should be in here. Just ocmenuitems